client/src/index.js

```diff
@@ -3,6 +3,11 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { fab } from '@fortawesome/free-brands-svg-icons'
+import { faCheckSquare, faUser, faBell} from '@fortawesome/free-solid-svg-icons'
+
+library.add(fab, faCheckSquare, faUser, faBell)
```

Comment:
Did you confirm with the design team before going with fa icons?

Reply:
Yes it's fine

Reply:
I will need you to change this to the individual use approach, we will not be using Global use approach, you can read here for the advantages or disadvantages of both https://fontawesome.com/v5.15/how-to-use/on-the-web/using-with/react#using